Kitodo-DataAccess/src/test/java/org/kitodo/dataaccess/NoDataExceptionTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataaccess;
+
+import org.junit.Test;
+import org.kitodo.dataaccess.NoDataException;
+
+public class NoDataExceptionTest {
+
+    @Test(expected = NoDataException.class)
+    public void testNoDataExceptionCanBeThrown() throws NoDataException {
+        throw new NoDataException();
```

Comment:
What this kind of test? You are testing the test framework!

Reply:
Tests that NoDataException can be thrown (i.e., does not throw another exception by itself).

Reply:
But it is like testing if Java language works...

Reply:
This is true for all tests. I think it should not be that an exception constructor throws an exception by itself, and it would be a bug that is hard to find because the exception ‘almost’ works (it still breaks the code flow, just a corresponding `catch` will not be invoked.)